geode-benchmarks/src/test/java/org/apache/geode/benchmark/tests/GeodeBenchmarkTest.java

```diff
@@ -36,46 +36,66 @@
  */
 class GeodeBenchmarkTest {
 
+  public static final String WITH_CLUSTER_TOPOLOGY = "withClusterTopology";
+  public static final String WITH_SNI_PROXY = "withSniProxy";
   private TestConfig config;
   private TestStep startProxyStep;
 
   @BeforeEach
   public void beforeEach() {
+    System.clearProperty(WITH_SNI_PROXY);
+    System.clearProperty(WITH_CLUSTER_TOPOLOGY);
```

Comment:
The old `@AfterAll` prevented subsequent tests run in the same JVM from seeing these system properties. This approach leaves the system properties in effect after the test finishes.

Is that important? Seems like it might be good to be sure and clear those in an `@AfterAll`.

Reply:
I'll put that back - thanks Bill